### PR TITLE
Sync consumer balance every time on startup

### PR DESCRIPTION
**Pull request by Waldz**
Closes: #1851

### CODE REVIEW

File: session/pingpong/invoice_payer.go [hunk 1]
```diff
@@ -154,24 +159,7 @@ func (ip *InvoicePayer) Start() error {
 	}
 }
 
-const grandTotalKey = "consumer_grand_total"
-
-func (ip *InvoicePayer) getGrandTotalPromised() (uint64, error) {
```
Comment (vkuznecovas) [inline session/pingpong/invoice_payer.go]: don't remove this func, the force-sync is temporary and we will need to operate on balances we have in state

Comment (Waldz) [reply session/pingpong/invoice_payer.go]: So we have `ip.deps.ConsumerTotalsStorage.Get()` for that and I created `recoverGrandTotalPromised()` for syncing it if needed

File: session/pingpong/invoice_payer.go [hunk 2]
```diff
@@ -204,12 +192,16 @@ func (ip *InvoicePayer) recoverGrandTotalPromised() (uint64, error) {
 		return nil
 	}
 
-	err := backoff.Retry(toRetry, boff)
-	if err != nil {
-		log.Error().Err(err).Msg("could not get balance from accountant")
```
Comment (zolia) [inline session/pingpong/invoice_payer.go]: Why to remove more detailed description here? We could say that "Failed after retry".

Comment (Waldz) [reply session/pingpong/invoice_payer.go]: It's moved one layer up

**Pull request merged**